app_dart/bin/server.dart

```diff
@@ -139,6 +139,24 @@ Future<void> main() async {
       ///
       /// Response: Status 200 OK
       '/api/update-task-status': UpdateTaskStatus(config, swarmingAuthProvider),
+
+      /// Apply task updates via PubSub messages.
```

Comment:
This doc isn't very sufaceable. If you add it as a class dartdoc in app_dart/lib/src/request_handlers/task_status_subscription.dart it'll show throughout the IDE references for it

Reply:
There are a lot of docs like this in the file. We should refactor them at once.

Reply:
I filed https://github.com/flutter/flutter/issues/97675